packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -99,6 +99,12 @@ export interface TransformConfig {
    * Such as sync configuration for appsync local support
    */
   ResolverConfig?: ResolverConfig;
+
+  /**
+   * List of custom transformer plugins
+   */
+  transformers?: string[];
+  warningESMessage?: boolean;
```

Comment:
Aren't we using Elasticsearch Warning for this? If we are changing it we might want to use a feature flag?